support/certs/tls.go

```diff
@@ -376,16 +367,20 @@ func ReconcileSignedCert(
 }
 
 type CAOpts struct {
-	CASignerCertMapKey string
-	CASignerKeyMapKey  string
+	CASignerCertMapKey  string
+	CASignerKeyMapKey   string
+	TLSSignerCertMapKey string
+	TLSSignerKeyMapKey  string
 }
 
 func (s *CAOpts) withDefaults() *CAOpts {
 	if s.CASignerCertMapKey == "" {
```

Comment:
why does CASignerCertMapKey dictates both CASignerCertMapKey / TLSSignerKeyMapKey?

Reply:
TLSSignerCertMapKey is setup to mirror CASignerCertMapKey, and TLSSignerKeyMapKey is set to mirror CASignerKeyMapKey.

Originally, I was just change the const values for CASignerCertMapKey, CASignerKeyMapKey; however @csrwng said that would probably be too disruptive, especially if there are existing clusters that need to be upgraded (as in IBM's case). Basically we need to change the CA reconciliation to write the same data to the keys asked for in the Jira ticket.

Reply:
Oh I see. Yeah that makes sense. It's been updated.